app/views/layouts/application.html.erb

```diff
@@ -1,27 +1,41 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Acebook</title>
-    <%= csrf_meta_tags %>
+<head>
+  <title>Acebook</title>
+  <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all' %>
-    <%= javascript_include_tag 'application' %>
-  </head>
+  <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'application' %>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
 
-  <body>
-    <% if signed_in? %>
-      Signed in as: <%= current_user.email %>
-      <%= button_to 'Sign out', sign_out_path, method: :delete %>
-    <% else %>
-      <%= link_to 'Sign in', sign_in_path %>
-    <% end %>
+<body>
+  <nav>
+    <a href="/">
+      <div id="page-logo">
+      </div></a>
+      <% if signed_in? %>
+      <div class="dropdown">
+        <li class="dropbtn"><i class="fa fa-chevron-circle-down" style="font-size:20px"></i></li>
```

Comment:
Hi Kay! We actually tried this first and it didn't work that time, though we didn't think to add an extra class just for styling icons. I've just changed it now and it still wasn't happy to co-operate unless we appended '!important' to the end to override any other styling....though we aren't sure where the other styling is coming from! Perhaps it's part of the external Font Awesome formatting (where we got the icon from)?